broker_app.py

```diff
@@ -46,9 +46,9 @@
 
 SPREADSHEET_UPLOAD_MESSAGE_ERROR = "We experienced a problem while uploading your spreadsheet"
 
-ingest_api = IngestApi()
-spreadsheet_generator = SpreadsheetGenerator(ingest_api)
-spreadsheet_job_manager = SpreadsheetJobManager(spreadsheet_generator, SPREADSHEET_STORAGE_DIR)
+global ingest_api
```

Comment:
Is using `global` a safe approach?
Should we use flask's app context instead? or the "g" object?

Reply:
Ah, i didn't know about that. Applied.

Reply:
Hmm it doesn't function as a global variable.

> This is a good place to store resources during a request. During testing, you can use the Faking Resources and Context pattern to pre-configure such resources.

I'll revert to using the global :(

Reply:
It might be my lack of knowledge, but why are we defining these 3 variables here and once again at the `setup` method?